docusaurus/docs/how-to-guides/data-source-plugins/add-authentication-for-data-source-plugins.md

```diff
@@ -311,14 +311,14 @@ Be aware that `tokenAuth` configuration is only supported in data source plugins
 
 :::
 
-## Authenticate using a backend plugin
+## Authenticate using a plugin backend 
 
 While the data source proxy supports the most common authentication methods for HTTP APIs, using proxy routes has a few limitations:
 
 - Proxy routes only support HTTP or HTTPS.
 - Proxy routes don't support custom token authentication.
 
-If any of these limitations apply to your plugin, you need to add a [backend plugin](../../key-concepts/backend-plugins/#caching-and-connection-pooling). Because backend plugins run on the server, they can access decrypted secrets, which makes it easier to implement custom authentication methods.
+If any of these limitations apply to your plugin, you need to add a [plugin backend component](../../key-concepts/backend-plugins/#caching-and-connection-pooling). Because plugin backend components run on the server, they can access decrypted secrets, which makes it easier to implement custom authentication methods.
```

Comment:
```suggestion
If any of these limitations apply to your plugin, you need to add a [backend component](../../key-concepts/backend-plugins/#caching-and-connection-pooling). Because plugin backend components run on the server, they can access decrypted secrets, which makes it easier to implement custom authentication methods.
```